backend/entityservice/database/util.py

```diff
@@ -54,11 +54,11 @@ def init_db_pool():
 
     global connection_pool
     if connection_pool is None:
-        logger.info("Initialize the database connection pool.", database=db, user=user, password=pw, host=host)
+        logger.info("Initialize the database connection pool.")
```

Comment:
why do you delete the extra info? I agree that revealing the password might not be such a great idea, but the rest might actually be helpful in some cases...

Reply:
:+1: